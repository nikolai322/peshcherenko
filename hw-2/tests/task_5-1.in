3
1 2 3
4
1.0 0.5 0.5 1.5
