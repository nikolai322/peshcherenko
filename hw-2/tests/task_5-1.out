6
3.500000
